sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultBackupClient.cs

```diff
@@ -79,13 +74,17 @@ public virtual async Task<KeyVaultBackupOperation> StartBackupAsync(Uri blobStor
             scope.Start();
             try
             {
-                var response = await _restClient.FullBackupAsync(
-                    VaultUri.AbsoluteUri,
+                var operation = await _restClient.FullBackupAsync(
+                    WaitUntil.Started,
                     new SASTokenParameter(blobStorageUri.AbsoluteUri, sasToken),
                     cancellationToken)
                     .ConfigureAwait(false);
 
-                return new KeyVaultBackupOperation(this, response);
+                // Rest client returns an Operation without headers, so we need to create a new response with headers.
```

Comment:
should we mention the generator issue here that, when implemented, indicates when we can remove this custom code?

Reply:
I don't find it necessary because when it gets implemented this is likely to break, so it would catch the devs attention.